image/moe/config.py

```diff
@@ -5,11 +5,8 @@
 NUM_EPOCHS = 200
 WARMUP_EPOCHS = 40
 
-WORLD_SIZE = 4
-MOE_MODEL_PARALLEL_SIZE = 1
-
-parallel = dict(
-    moe=dict(size=MOE_MODEL_PARALLEL_SIZE)
-)
+parallel = dict()
+max_ep_size = 1  # all experts are replicated in the case that user only has 1 GPU

```

Comment:
can not understand the comments? 
You mean there is only 1 expert and it is duplicated on each GPU?

Reply:
please explain the meaning of max_ep_size. Better with an example.